service/crt/service.go

```diff
@@ -29,6 +30,11 @@ const (
 	// resyncPeriod is the period for re-synchronizing the list of objects in k8s
 	// watcher. Set to 1 minute to make the watch more robust.
 	resyncPeriod time.Duration = time.Minute * 1
+
+	TPRName        = "certificate"
```

Comment:
@xh3b4sd I don't think the abstraction is wrong. My thinking was to make it possible to create list/watch URLs from these values. While cert-operator uses certificatetpr for TPR it has watch/list URLs baked in. Which I find even worse bc it gives false feeling it's configureable.